contracts/interfaces/AddressBookInterface.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+interface AddressBookInterface {
+    function getOtokenImpl() external view returns (address otoken);
+
+    function getWhitelist() external view returns (address whitelist);
```

Comment:
question on this @haythem96 for return args should we do (type name) or just (type) ? Also how should the name be formatted?

Reply:
I'm okay with either in interfaces. But in actual functions, I like having names for the return variables, it helps a lot in terms of readability when you return more than 1 parameters.

Reply:
Agree with Anton.